xtext-website/documentation/350_continuous_integration.md

```diff
@@ -307,6 +307,7 @@ To further speed up the p2 dependency resolution step, use the concrete build re
 
 | Xtext         | EMF           | MWE2/MWE    | Xpand       | Eclipse     | All included in |
 | ------------- | ------------- | ----------- | ----------- | ----------- | ----------- |
+| [2.33.0]({{page.upsite.xtext}}releases/2.33.0/)           | [2.36.0]({{page.upsite.eclipse}}modeling/emf/emf/builds/release/2.36.0) (2.29.0)     | [2.15.0]({{page.upsite.mwe}}releases/2.15.0/) (2.9.1) | no longer supported  | [4.20.0]({{page.upsite.eclipse}}releases/2023-12) (4.23) | [2023-12]({{page.upsite.eclipse}}releases/2023-12)|
```

Comment:
emf version needs bumping

Reply:
mwe or emf? or both?

Reply:
mwe

Reply:
EMF is 2.36 which is correct 
 https://github.com/eclipse-simrel/simrel.build/pull/92
 had already preliminary adjusted that one.
for mwe did not anticipate new release